package.json

```diff
@@ -23,14 +23,17 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "npm run lint && mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test": "npm run lint && npm run test-node && npm run test-electron",
+    "test-node": "mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test-electron": "electron-mocha --recursive tests/*.spec.js -R spec",
```

Comment:
Any reason for `tests/e2e.js` to be included in `test-node` but not `test-electron`? Or is that another one of my stupid questions? ;-)